prime-router/src/main/kotlin/common/HttpClientUtils.kt

```diff
@@ -32,10 +30,30 @@ class HttpClientUtils {
         const val REQUEST_TIMEOUT_MILLIS: Long = 130000 // need to be public to be used by inline
         const val SETTINGS_REQUEST_TIMEOUT_MILLIS = 30000
 
+        private val httpClient: HttpClient =
+            HttpClient(Apache) {
+                install(ContentNegotiation) {
+                    json(
+                        Json {
+                            prettyPrint = true
+                            isLenient = true
+                            ignoreUnknownKeys = true
+                        }
+                    )
+                }
+                install(HttpTimeout)
+                engine {
+                    followRedirects = true
+                    socketTimeout = TIMEOUT
+                    connectTimeout = TIMEOUT
+                    connectionRequestTimeout = TIMEOUT
```

Comment:
Any use in making these values different or is `TIMEOUT` just the max length for any kind of timeout?

Reply:
I left it set to what it was set prior. If there's something you'd like me to set one or all of these too I'm open to suggestion